README.md

```diff
@@ -97,222 +98,164 @@ langstar prompt get owner/prompt-name
-
-```bash
-# List all deployments
-langstar graph list
+# Search for assistants by name
+langstar assistant search "customer-service"
 
-# Output example:
-# Name                    ID                   Status  Created
-# my-prod-deployment      abc-123e4567...      READY   2024-01-15
-# my-staging-deployment   def-456e4567...      READY   2024-01-20
-```
+# Get details of a specific assistant
+langstar assistant get <assistant-id>
 
-You can filter and search:
+# Create a new assistant
+langstar assistant create --graph-id <graph-id> --name "My Assistant"
 
-```bash
-# Filter by deployment type
-langstar graph list --deployment-type prod
+# Create with configuration
+langstar assistant create --graph-id <graph-id> --name "Configured Bot" \
+  --config '{"temperature": 0.7}'
 
-# Filter by status
-langstar graph list --status READY
+# Update an assistant
+langstar assistant update <assistant-id> --name "Updated Name"
 
-# Filter by name
-langstar graph list --name-contains "production"
+# Delete an assistant
+langstar assistant delete <assistant-id>
+langstar assistant delete <assistant-id> --force  # Skip confirmation
 
-# JSON output for scripting
-langstar graph list --format json
+# JSON output
+langstar assistant list --format json
 ```
 
-### Managing Assistants
+## Configuration
 
-All assistant commands require the `--deployment` flag to specify which deployment to target. You can use either the deployment **name** or **ID** from `langstar graph list`.
+This section provides detailed configuration options for both LangSmith and LangGraph services.
 
-#### List Assistants
+### Configuration Methods
 
-```bash
-# List assistants in a deployment (by name)
-langstar assistant list --deployment my-prod-deployment
+Langstar supports three configuration methods, in order of precedence:
 
-# Or by ID
-langstar assistant list --deployment abc-123e4567
+1. **Command-line flags** (highest priority)
+2. **Configuration file** (`~/.langstar/config.toml`)
+3. **Environment variables** (lowest priority)
 
-# With pagination
-langstar assistant list --deployment my-prod-deployment --limit 10 --offset 20
+### Configuration File Format
 
-# JSON output
-langstar assistant list --deployment my-prod-deployment --format json
-```
+Create a configuration file at `~/.langstar/config.toml`:
 
-#### Search Assistants
+```toml
+[langstar]
+# Output format (table or json)
+output_format = "table"
 
-```bash
-# Search by name
-langstar assistant search "customer" --deployment my-prod-deployment
+# LangSmith configuration (for prompt commands)
+langsmith_api_key = "<your-langsmith-key>"
+organization_id = "<your-org-id>"        # Optional: scope to organization
+workspace_id = "<your-workspace-id>"     # Optional: scope to workspace
 
-# With result limit
-langstar assistant search "bot" --deployment my-staging --limit 5
+# LangGraph configuration (for assistant commands)
+langgraph_api_key = "<your-langgraph-key>"  # No scoping needed
```

Comment:
again, we should not use a different api key for langgraph and langsmith. In fact langgraph cloud is now langsmith deployments (they have rebranded)